dist/assets/js/nominatim-ui.js

```diff
@@ -757,7 +757,10 @@ jQuery(document).ready(function () {
 
   function parse_url_and_load_page() {
     // 'search', 'reverse', 'details'
-    var pagename = window.location.pathname.replace('.html', '').replace(/^\//, '');
+    var pagename = window.location.pathname.replace('.html', '').replace(/.+\//, '');
+
+    if (pagename === '') pagename = 'search'
+    console.log(pagename)
```

Comment:
Debug leftover?

Reply:
Removed now. 'yarn lint' even printed it as error, I should add CI tests